### PR TITLE
Nesting inside stack 2 - new

### PR DESCRIPTION
**Pull request by anjalirana**
Nesting inside stack 2 - new

### CODE REVIEW

File: Packer.php [hunk 1]
```diff
@@ -205,11 +205,12 @@ public function doVolumePackingWithLimitedBoxes() {
     $boxesToEvaluate = clone $this->boxes;
 
     while (!$boxesToEvaluate->isEmpty()) {
```
Comment (beefsandwich) [inline Packer.php:207]: What's the purpose of the changes in this file?  Is there a corresponding test?

**Review by beefsandwich — APPROVED** [review]
All good

File: Packer.php (hunk 1 above)
Comment (anjalirana) [reply Packer.php:207]: So if none of the boxes fit any of the items in it when the boxes are limited then it should not throw an error rather it should just return an empty packedBoxList - for shipping nesting purposes.
